frontend/src/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import logo from "./assets/logo.svg";
+export function Navbar() {
+  return (
+    <>
+      <div className="videonavbar">
+        <div className="videonavbar-logo">
+          <img src={logo} alt="logo" />
+          <span>K</span>rishiGyan
+        </div>
```

Comment:
When click happens on logo and title it does not route to home page so that can be done which is uniform across all websites.